### ISSUE TITLE
Is it possible to exclude fields from the model itself when exporting them? (instead of dict(exclude=..))

### ISSUE THREAD

**Issue by arielb135** [samuelcolvin/pydantic#1862]
Basically, i have the following model:
```
class MyCls(BaseModel):
    permissions: Permissions
    some_id: str = Field('')
    some_flag: bool = Field(False)

```
when exporting this module to dict / anything else - i want to exclude some_flag from the output.
I know it can be done through the export in the dict method - but this class is a subclass in a more complex model, and i don't want the user of the class to decide what to exclude.

Note - i'm using this flag for some inner calculations.

**Comment by PrettyWood** [comment]
Hello @arielb135 
Currently it's not possible without overriding `dict` method (or any other workaround).
But I think [support of private attributes](https://github.com/samuelcolvin/pydantic/pull/1679) or having a special value of [dump alias](https://github.com/samuelcolvin/pydantic/pull/1695) (like `dump_alias=None`) to exclude fields would be two viable solutions.
Both solutions may be included in pydantic 1.7 if everything goes well.
WDYT ?

**Comment by daviskirk** [comment]
I think this might be a duplicate of https://github.com/samuelcolvin/pydantic/issues/660 right?
The solution (I think ... when that's done) would look something like:

```python
class MyCls(BaseModel):
    permissions: Permissions
    some_id: str = Field('')
    some_flag: bool = Field(False)

    class Config:
        fields = {
            'some_flag': {
                'exclude': ...,
            }
        }
```
